src/librustdoc/html/render/context.rs

```diff
@@ -22,7 +22,7 @@ use super::{
     BASIC_KEYWORDS,
 };
 
-use crate::clean;
+use crate::{clean, try_err};
```

Comment:
nit: It feels like these should be on separate lines. Maybe also merge together `use crate::clean` and `use crate::clean::ExternalCrate`?